apps/web/utils/actions/group.ts

```
@@ -115,21 +206,71 @@ export async function createReceiptGroupAction(): Promise<
+    if (!user) return { error: "User not found" };
+
+    await generateGroupItemsFromPrompt(
+      groupId,
+      user,
+      gmail,
+      token.token,
+      existingGroup.name,
+      existingGroup.prompt,
+    );
+    return;
+  }
```

Biome

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)